sunpy/sun/sun.py

```diff
@@ -20,42 +43,6 @@
 Carrington Rotation Number = 1971.4091        check!
```

Comment:
Has this been checked? Also is there a reason this isn't with the rest of the docstring?